gdal_subsetter/transform.py

```diff
@@ -353,57 +317,14 @@ def update_layernames(self, file_name, layer_names):
 
         dataset = None
 
-    def prepare_output_dir(self, output_dir):
```

Comment:
This method was only being called in a test (and that test is currently being skipped anyway). I've updated the test to remove the output directory if it exists there, rather than including that code in this class.